engines/sky/logic.cpp

```diff
@@ -1723,10 +1723,6 @@ bool Logic::fnSpeakWaitDir(uint32 a, uint32 b, uint32 c) {
 	b is text message number
 	c is base of mini table within anim_talk_table */
 
-#ifdef __DC__
-	__builtin_alloca(4); // Works around a gcc bug (wrong-code/11736)
```

Comment:
This bug got closed with GCC 4.7 (https://gcc.gnu.org/bugzilla/show_bug.cgi?id=11736) which supports C++11.
It is safe to remove though because we are now using GCC 4.9.4 for Dreamcast

Reply:
Yes, I refer to it in the associated commit ;) GCC 4.7 wouldn't have enough C++11 support anyway, I believe.

Thanks for the toolchain confirmation